### PR TITLE
add documentation on the withAnyMessage and withMessage methods of the error asserter

### LINKED ISSUE #154: error asserter : missing documentation on withAnyMessage and withMessage

### PR DESCRIPTION
**Pull request by agallou**
fixes #154

### CODE REVIEW

File: source/en/asserters/error.inc.rst [hunk 1]
```diff
@@ -113,3 +113,78 @@ withType
            ->exists()
    ;
 
+
+.. _with-message:
+
+withMessage
+===========
+
+``withMessage`` checks the message's content of the raised error message.
```
Comment (jubianchi) [inline source/en/asserters/error.inc.rst]: `s/of the raised error message/of the raised error/`

**Comment by jubianchi** [comment]
:+1:

File: source/en/asserters/error.inc.rst [hunk 2]
```diff
@@ -113,3 +113,78 @@ withType
            ->exists()
    ;
 
+
+.. _with-message:
+
+withMessage
+===========
+
+``withMessage`` checks the message's content of the raised error.
```
Comment (marmotz) [inline source/en/asserters/error.inc.rst]: The message content

File: source/en/asserters/error.inc.rst [hunk 3]
```diff
@@ -113,3 +113,78 @@ withType
...
+           ->exists() // fails
+   ;
+
+
+.. _with-any-message:
+
+withAnyMessage
+==============
+
+``withAnyMessage`` does not check the error message. That's the default  behaviour. So ``->error()->withAnyMessage()->exists()`` is the equivalent of ``->error()->exists()``. This method is here if you want to add sematic to your test.
```
Comment (marmotz) [inline source/en/asserters/error.inc.rst]: Semantic

File: source/en/asserters/error.inc.rst [hunk 4]
```diff
@@ -113,3 +113,78 @@ withType
...
+
+   <?php
+   $this
+       ->when(
+           function() {
+               trigger_error('message');
+           }
+       )
+       ->error()
+           ->withMessage("message")
```
Comment (marmotz) [inline source/en/asserters/error.inc.rst]: Unconsistent simple/double quote between this line and line 130.
Please use simple quote

File: source/en/asserters/error.inc.rst [hunk 5]
```diff
@@ -113,3 +113,78 @@ withType
...
+   ;
+
+   $this
+       ->when(
+           function() {
+               trigger_error('message');
+           }
+       )
+       ->error()
+           ->withMessage("MESSAGE")
```
Comment (marmotz) [inline source/en/asserters/error.inc.rst]: Unconsistent simple/double quote between this line and line 141.
Please use simple quote

File: source/fr/assertions/error.inc.rst [hunk 6]
```diff
@@ -113,3 +113,79 @@ withType
...
+
+   <?php
+   $this
+       ->when(
+           function() {
+               trigger_error('message');
+           }
+       )
+       ->error()
+           ->withMessage("message")
```
Comment (marmotz) [inline source/fr/assertions/error.inc.rst]: Unconsistent simple/double quote between this line and line 131.
Please use simple quote

File: source/fr/assertions/error.inc.rst [hunk 7]
```diff
@@ -113,3 +113,79 @@ withType
...
+   ;
+
+   $this
+       ->when(
+           function() {
+               trigger_error('message');
+           }
+       )
+       ->error()
+           ->withMessage("MESSAGE")
```
Comment (marmotz) [inline source/fr/assertions/error.inc.rst]: Unconsistent simple/double quote between this line and line 142.
Please use simple quote

File: source/fr/assertions/error.inc.rst [hunk 8]
```diff
@@ -113,3 +113,79 @@ withType
...
+           ->exists() // échoue
+   ;
+
+
+.. _with-any-message:
+
+withAnyMessage
+==============
+
+``withAnyMessage`` ne vérifie pas le message de l'erreur. C'est le comportement par défaut de l'asserter. Donc ``->error()->withAnyMessage()->exists()`` est l'équivalent de ``->error()->exists()``. Cette méthode existe pour ajouter de la sématique dans vos tests.
```
Comment (marmotz) [inline source/fr/assertions/error.inc.rst]: Sémantique

**Comment by agallou** [comment]
@marmotz documentation updated.

**Comment by marmotz** [comment]
Thank you!

**Pull request merged**